backend/node/genvm/base.py

```diff
@@ -141,7 +157,10 @@ async def run_contract(
         execution_result = ExecutionResultStatus.SUCCESS
         error = None
 
-        with safe_globals():
+        # Buffers to capture stdout and stderr
+        stdout_buffer = io.StringIO()
+
+        with redirect_stdout(stdout_buffer), safe_globals():
```

Comment:
should we also redirect stderr?

Reply:
I tried but even the errors were redirected to the stdout. Probably because they are handled by the Exception and printed.